pages/community/community-projects.md

```diff
@@ -1,32 +0,0 @@
----
```

Comment:
You probably did not want to remove this file. You can easily fix this: https://stackoverflow.com/a/900563/2254346